pkg/controller/pv.go

```diff
@@ -236,3 +236,15 @@ func NewPVController(mgr manager.Manager, azVolumeClient azVolumeClientSet.Inter
 	klog.V(2).Info("Controller set-up successful.")
 	return &reconciler, nil
 }
+
+func (r *ReconcilePV) getClient() client.Client {
```

Comment:
Are these methods used anywhere?

Reply:
These methods are needed for PV controller to implement `azReconciler` interface.